docs/adr/004-database.md

```diff
@@ -0,0 +1,26 @@
+
+# 4. Database 
+
+Date: 2021-06-25
+
+## Status
+
+Accepted
+
+## Context
+
+We want to pick a database for Unicorn development.
+
+We will likely need a relational database since the data is mostly structured and related.
+
+Reference to [different types of databases and their usages](https://www.geeksforgeeks.org/types-of-databases/).
+
+## Decision
```

Comment:
Can we add a section for options considered, and maybe key factors considered?

Not looking to have you extend your analysis in any way, just that it would be helpful to be more explicit about what ended up being your primary differentiators (e.g., open-source vs proprietary and TTS experience) and what you did/did not consider.